go1.x-image/cookiecutter-aws-sam-hello-golang-lambda-image/{{cookiecutter.project_name}}/hello-world/Dockerfile

```diff
@@ -1,13 +1,8 @@
-FROM public.ecr.aws/bitnami/golang:1.16 as build-image
-
-WORKDIR /go/src
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as build-image
+RUN yum install -y golang
```

Comment:
Would it be possible to use the `1.18` tagged Bitnami golang image over AL2 (ie. `public.ecr.aws/bitnami/golang:1.18`)? The concern here is that the initial time it takes to install golang and it's dependencies with that image isn't ideal from the point of view of a new user that wants to try out golang.

Reply:
I think that'll lock out arm image builds 

from: https://gallery.ecr.aws/bitnami/golang
> OS/Arch: Linux, x86-64


What do you think about using https://gallery.ecr.aws/docker/library/golang instead?

Reply:
That looks like a possible solution, lets try using those images instead